src/Administration/Resources/app/administration/src/module/sw-settings-payment/page/sw-settings-payment-detail/index.js

```diff
@@ -164,7 +164,7 @@ export default {
                 .then((paymentMethod) => {
                     this.paymentMethod = paymentMethod;
 
-                    if (!paymentMethod || !hasOwnProperty(paymentMethod, 'mediaId')) {
+                    if (!paymentMethod || !hasOwnProperty(paymentMethod, 'mediaId') || !paymentMethod.mediaId) {
```

Comment:
```suggestion
                    if (!paymentMethod || !hasOwnProperty(paymentMethod, 'mediaId') || !paymentMethod.mediaId) {
```
`if (!paymentMethod?.mediaId) {`